metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/tree_function.py

```diff
@@ -28,13 +32,14 @@ def get_variable_statement(parse_tree: Tree, variable: str) -> Optional[Tree]:
     for tree in _filter:
         values: List[str] = token_values(tree.children[0])
         actual_value: str = "".join(strip_char_from_list(values, " "))
-        logger.debug(f"Actual Value = {actual_value}")
-        logger.debug(f"Expected Value = {variable}")
+        if TRACE_POWERBI_MQUERY_PARSER:
+            logger.debug(f"Actual Value = {actual_value}")
```

Comment:
do we want a higher level here if tracing is enabled?

Reply:
if they have trace enabled, they probably should have debug too. I expect the trace stuff will only be useful in dev so it should be fine